### ISSUE TITLE
New Beast Sheets Having 1 Caster Level by Default

### ISSUE THREAD

**Issue by Smothmoth** [n0q/hackmaster5e-fvtt#305]
Whenever you create a new actor with the Beast sheet, they always have '1' pre entered into the Caster Levels field by default. This is hardly a serious problem and has been present for a very long time, yet I only just considered that perhaps it wasn't intended to be that way.

**Comment by n0q** [comment]
Yes, that's intentional. It's presumed that if a beast has spells to cast, you at least intended for it to have a caster level. If they aren't casters, then the 1 doesn't hurt. I could blank it out by default if it seems out of place.

**Comment by Smothmoth** [comment]
It does seem out of place to me but its hardly a big deal especially if it doesn't actually impact anything, I've always just blanked it out myself whenever making a new NPC that isn't a caster.
